### DISCUSSION TITLE
v0.12.5

### DISCUSSION THREAD

**Discussion by AsamK** [AsamK/signal-cli#1388] [Releases]
[Changelog](https://github.com/AsamK/signal-cli/blob/master/CHANGELOG.md#0125---2023-11-21)

**Full Changelog**: https://github.com/AsamK/signal-cli/compare/v0.12.4...v0.12.5

<hr /><em>This discussion was created from the release <a href='https://github.com/AsamK/signal-cli/releases/tag/v0.12.5'>v0.12.5</a>.</em>

**Comment by cumulomeraviglio** [comment]
Hi AsamK
I tried right now to install again this new version, linking again on that computer was not working.
Unfortunately terminal tells me that the version of Java I have is too new for signal-cli.
I use ubuntu 22, and was signal with siagnal-cli, from now, with no issues.
Does anyone have any idea how to fix this ( and maybe all the conversations back after ?)
cheers

**Reply by cumulomeraviglio** [reply]
To be more clear !
I deleted signal-cli-0.9.5, downloaded the new one, tried to register again with the same number.
There came this message about java.

**Reply by AsamK** [reply]
This version just needs java 17
